web/src/app/layout.tsx

```diff
@@ -0,0 +1,17 @@
+"use client";
```

Comment:
Your root layout should remain a server component (it's okay that your providers, which are client components, are children here)

Reply:
This was what was confusing me a lot early on, I'm looking forward to Chakra making SSR and progressive enhancement easier in future versions but this works for now 🎉